src/ansys/fluent/core/file_session.py

```diff
@@ -105,6 +105,13 @@ def add_scalar_fields_request(
         Returns
         -------
         None
+
+        Raises
+        ------
+        RuntimeError
+            If surface names or surface ids are not provided.
```

Comment:
I believe "id" should be uppercase as "ID", but I think we already use it in too many places as "id" to change this in this PR, better do this in a separate PR if anyone considers this an actual issue in the future

Reply:
I agree with @raph-luc, especially as id (plural ids) is a noun.